api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -161,3 +161,15 @@ message GrpcJsonTranscoder {
   //  :ref:`proto descriptor set <config_grpc_json_generate_proto_descriptor_set>`.
   bool convert_grpc_status = 9;
 }
+
+message GrpcJsonTranscoderPerRoute {
+  oneof override {
+    option (validate.required) = true;
+
+    // Disable the grpc json trasncoder filter for this particular vhost or route.
+    bool disabled = 1 [(validate.rules).bool = {const: true}];
```

Comment:
I know we do the `const: true` thing elsewhere in the API, but not sure what it actually does?

Reply:
as far as i understand - it makes sure that the validator will check that the value of `disabled` is exactly `true`. (since it's a oneof, it doesn't make sense for it to be false anyways)